src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -649,6 +649,12 @@ static ucs_status_t uct_ib_mlx5dv_md_open(struct ibv_device *ibv_device,
         goto err_free;
     }
 
+#if HAVE_DECL_IBV_ODP_SUPPORT_IMPLICIT
+    if (dev->dev_attr.odp_caps.general_caps & IBV_ODP_SUPPORT_IMPLICIT) {
+        dev->flags |= UCT_IB_DEVICE_FLAG_ODP_IMPLICIT;
```

Comment:
should we get rid of uct_ib_device_odp_has_global_mr() and make some common function to be used by uct_ib_verbs_md_open() as well?